app/views/templates/rulingCertificateTemplate.scala.html

```diff
@@ -20,33 +20,28 @@
 @(appConfig: FrontendAppConfig, c: Case, d: Decision, getCountryName: String => Option[String],
 compositeMode: Boolean = false)(implicit messages: Messages)
 
-@createLink(path: String)(implicit req: play.api.mvc.Request[_]) = @{
-val protocol = if(req.secure) "https" else "http"
-s"$protocol://${req.host}${routes.Assets.versioned(path)}"
-}
-
-<div id="print-pages" class="print-pages">
-    <div id="ruling-certificate" class="print-document" id="print-document">
+<div class="print-pages">
+    <div class="print-document" id="print-document">
         <div class="govuk-grid-row print-header">
             <div class="govuk-grid-column-one-third">
                 <img class="crown-logo-atar" alt="HM Revenue and Customs" src="@messages("logo.encoded")"/>
             </div>
             <div>
-                <h2 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
+                <h1 class="govuk-heading-m margin-top-small no-margin-bottom print-header__heading">
```

Comment:
please revert back to <h2> to pick up styling

Reply:
done